test/utils/utils.testSuite.ts

```diff
@@ -2,6 +2,7 @@ import { changeResolverTests } from "./changeResolver.testSuite";
 import { didTests } from "./did.testSuit";
 
 export const utilsTests = () => {
-    describe("Change resolver tests", changeResolverTests);
+    // Skipped until either changeResolver script is changed according with Public Resolver designe or there will be no sence in migration
```

Comment:
Thanks for the comment here. Could you elaborate a bit on what would need to be changed please?

Reply:
Ok. Added comment to `change_resolver`